src/tools/cephfs/top/cephfs-top

```diff
@@ -253,8 +272,9 @@ class FSTop(object):
             return False
         client_metadata = stats_json[CLIENT_METADATA_KEY]
         num_clients = len(client_metadata)
-        num_mounts = len([client for client, metadata in client_metadata.items() if not
-                          metadata[CLIENT_METADATA_MOUNT_POINT_KEY] == 'N/A'])
+        num_mounts = len([client for client, metadata in client_metadata.items() if
+                          (CLIENT_METADATA_MOUNT_POINT_KEY in metadata and
+                           metadata.get(CLIENT_METADATA_MOUNT_POINT_KEY) != 'N/A')])
```

Comment:
I think I left a comment last time for this (regarding tracking such "unknown" clients).